2.train-VAE/optimize_hyperparameters.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[11]:
```

Comment:
Please ensure the notebook is ran sequentially prior to converting to a script :D